plugins/other/pacman

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
You may want to switch from `bash` to `sh`, since you are using only two trivial bash-specific operators, which can be easily replaced (`[[ ... ]]` -> `[ ... ]` and `&>` -> `>/dev/null 2>&1`).
Is `bash` considered to be "essential" in an arch system?
Anyway - it is probably just a matter of taste ...